ci/bazel-scripts/main.sh

```diff
@@ -6,42 +6,39 @@
 
 set -eufo pipefail
 
+
+# default behavior is that it runs whatever targets were specified in BAZEL_TARGETS and uploads to S3
 ic_version_rc_only="0000000000000000000000000000000000000000"
-if [ "$CI_COMMIT_REF_PROTECTED" = "true" ]; then
-    ic_version_rc_only="${CI_COMMIT_SHA}"
-    s3_upload="True"
-fi
+s3_upload="True"
 
-if [[ "${CI_COMMIT_BRANCH:-}" =~ ^hotfix-.* ]]; then
```

Comment:
We can remove this hotfix branch logic because hotfix branches are protected and it's redundant.